website-next/components/projects/cards.tsx

```diff
@@ -1,6 +1,3 @@
-/** @jsx jsx */
-import React from 'react'
-import { jsx } from '@emotion/react'
```

Comment:
I'm surprised that removing these didn't cause any issues since this file uses jsx syntax

Reply:
oh I deleted these because at one point I had a very agressive linter enabled that would complain for unused vars. I think I'll put the react import back.

_by the way_ the file does not use JSX syntax! it only uses `styled` functions and style objects

Reply:
Ha I saw the `<CardProps>` below and mentally read it as a piece of JSX!